proto/bff/v1/moderation_service.proto

```diff
@@ -84,6 +87,12 @@ message ProcessApprovalQueueAuditPayload {
   string reason = 2;
 }
 
+message HoldBackPendingActorRequest {
```

Comment:
Let's make the duration configurable - it'll be easier for us to tweak the frontend than deploy the backend to adjust this.